homeassistant/components/fritz/__init__.py

```diff
@@ -1 +1,91 @@
-"""The fritz component."""
+"""Support for AVM Fritz!Box functions."""
+import logging
+
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_REAUTH, ConfigEntry
+from homeassistant.const import (
+    CONF_DEVICES,
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+)
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
+
+from .common import FritzBoxTools
+from .const import (
+    DATA_FRITZ_TOOLS_INSTANCE,
+    DEFAULT_HOST,
+    DEFAULT_PORT,
+    DOMAIN,
+    ERROR_CONNECTION_ERROR,
+    SUPPORTED_DOMAINS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
+    """Set up FRITZ!Box Tools component."""
+    if DOMAIN in config:
```

Comment:
```suggestion
    if DOMAIN not in config:
        return True
```

Then outdent the block below